### DISCUSSION TITLE
ARMAConv for Graph Clasification

### DISCUSSION THREAD

**Discussion by xnuohz** [danielegrattarola/spektral#175] [Q&A]
Hi,
Any ARMAConv examples for PPI and TU datasets reimplementation results in the original paper? Thanks:)

**Comment by danielegrattarola** [comment]
Hi, 

the code for the ARMA paper will be released on its own repo (I have asked the authors :D), but all datasets/tools available here should be sufficient to reproduce the results. 

Cheers
